packages/theme/src/color-ramps/lib/index.ts

```diff
@@ -204,14 +198,35 @@ function calculateRamp( {
 				! contrast.ignoreWhenAdjustingSeed && ! searchResults.reached,
 		};
 	}
-
 	return {
 		rampResults,
-		SATISFIED_ALL_CONTRAST_REQUIREMENTS,
-		UNSATISFIED_DIRECTION,
+		MAX_DEFICIT,
+		MAX_DEFICIT_DIRECTION,
+		MAX_DEFICIT_STEP,
 	};
 }
 
+// Return minimal set of steps that are needed to calculate `stepName` from the seed.
```

Comment:
Minor: For this to be associated in autocomplete, it should be TSDoc-formatted.

Reply:
Added a full tsdoc block for the function.